app/src/vanilla/java/cm/aptoide/pt/view/settings/NewAccountFragment.java

```diff
@@ -41,6 +44,8 @@
 import rx.Observable;
 import rx.android.schedulers.AndroidSchedulers;
 
+import static android.support.v4.content.ContextCompat.getDrawable;
```

Comment:
Personal preference: I would use ContextCompat.getDrawable(getContext(), drawableId))
directly instead of using a static import.